pkg/controller/migstorage/migstorage_controller.go

```diff
@@ -107,26 +108,33 @@ func (r *ReconcileMigStorage) Reconcile(request reconcile.Request) (reconcile.Re
 	err = r.Get(context.TODO(), request.NamespacedName, storage)
 	if err != nil {
 		if errors.IsNotFound(err) {
+			log.V(2).Info("unable to get migstorage, not found", "migstorage", request.String(), "error", err.Error())
 			return reconcile.Result{}, nil
 		}
-		log.Trace(err)
+		log.Trace(fmt.Errorf("unable to get migstorage: %#v", err), "migstorage", request.String())
```

Comment:
The `Logger.Trace()` is intended to simply log the error and stack trace.  Logger.Error("Unable to get MigStorage.", err) would be a better fit.
Same comment here re: configuring the Logging with the _storage_ keyvalues so we don't need to explicitly include it in every log message.

Reply:
+1